DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs

```diff
@@ -0,0 +1,35 @@
+namespace DotnetBoilerPlate.DAL.Models;
+
+public class CandidateExamsDBO
```

Comment:
We will not need this database object. We are trying to build a boilerplate app that has common work items that can be reused by copying this project content and building on top of it. So, not just the name, but anything that is related to Training pegasus will have to be removed.

Reply:
We should use Asp.Net Identity to setup logins/signups/roles. That will be a task on it's own.

Reply:
Removed all DBOs.Kept only Users files